app/views/videos/new.html.erb

```diff
@@ -33,7 +33,7 @@
   <% end %>
   <div style="margin:50px auto;max-width:80%;font-size:0.86em;">
     <p>
-      <strong>Video is beta:</strong> Email <%= email_link %> if you have any problems.
+      <strong>Video is beta:</strong> <%= t("contact_prompts.if_any_questions_html") %>
```

Comment:
Same here. "Video is beta:" isn't translated

Reply:
We might want to hold off on translating this file as its going to be changed by https://github.com/forem/forem/pull/14960